ROADMAP.md

```diff
@@ -28,4 +8,34 @@
  
 ## Other features we are working on in the 1.7 timeframe
 
+* etcd controller to allow moving between versions
 * kops server for better team scenarios
+* support for bare-metal
+* more gossip backends
+* IAM integration
+* more cloud providers
+* promote GCE to stable
```

Comment:
I was actually mentioning missing pieces in gce. Instance groups commands do not work ;)

Yes improvements to rolling-update are are on the way! Already have one pr.